tests/flytekit/unit/core/test_flyte_directory.py

```diff
@@ -140,8 +140,10 @@ def my_wf() -> FlyteDirectory:
     files = os.listdir(wfd.path)
     assert len(files) == 5
 
+    csv = str
+
     @task
-    def t2(in1: FlyteDirectory["csv"]) -> int:
+    def t2(in1: FlyteDirectory[csv]) -> int:
```

Comment:
It's a hack but could we use this style instead please https://github.com/flyteorg/flytekit/blob/b8a3c7d34e6b19722a6a968dc05506ad1eb26912/flytekit/types/file/__init__.py#L60?  It's a hack because it just so happens that the string representation is the string that you pass in, but whatever. generics in general are pretty hacky in python.

Reply:
maybe there is an alternative?